exercises/02-react-stateless/src/modules/root/components/header.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+interface HeaderProps {
+  title: string
+}
+
+export const Header: React.FC<HeaderProps> = ({ title }) => {
+  return <h1>{title}</h1>
+}
```

Comment:
Not necessary to change this but simple functions like this one don't need `return`.

```suggestion
export const Header: React.FC<HeaderProps> = ({ title }) => <h1>{title}</h1>
```